ledger/object/lock.go

```diff
@@ -53,13 +53,13 @@ func NewIndexLocker() IndexLocker {
 // If mutex does not exist, it will be created in concurrent safe fashion.
 func (l *idLocker) Lock(id insolar.ID) {
 	// Reset pulse. It should not be considered when locking.
-	id.SetPulse(0)
+	zeroID := *insolar.NewID(0, id.Hash())
```

Comment:
I think prefix `zero` is ineligible here. Actually it's not 'zero'-ID, it like ID without pulse prefix, it's better to call it like `normID` or smth like that